q2_dummy_types/plugin_setup.py

```diff
@@ -0,0 +1,23 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import importlib
+
+import qiime.plugin
+
+import q2_dummy_types
+
+plugin = qiime.plugin.Plugin(
+    name='dummy-types',
+    version=q2_dummy_types.__version__,
+    website='https://github.com/qiime2/q2-dummy-types',
+    package='q2_dummy_types'
+)
+
```

Comment:
If this is illustrative, could you add a comment explaining the importance of importing these modules in this file?